cluster/kubemark/gce/config-default.sh

```diff
@@ -39,7 +39,7 @@ NODE_OS_DISTRIBUTION=${KUBE_NODE_OS_DISTRIBUTION:-debian}
 MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-cos-stable-59-9460-64-0}
 MASTER_IMAGE_PROJECT=${KUBE_GCE_MASTER_PROJECT:-cos-cloud}
 
-NETWORK=${KUBE_GCE_NETWORK:-default}
+NETWORK=${KUBE_GCE_NETWORK:-e2e}
```

Comment:
Oh, no sorry, scratch that. It actually is a test cluster - it's kubemark setup.